Core/Src/main.c

```diff
@@ -44,6 +46,20 @@
 /* Private variables ---------------------------------------------------------*/
 
 /* USER CODE BEGIN PV */
+	const uint8_t FIRST_BYTE = 0b00000001; //send start bit
+	const uint8_t SECOND_BYTE = 0b10000000; //configure channel select CH0
+	const uint8_t THIRD_BYTE = 0b00000000; //all don't cares to get back 8bit register
```

Comment:
Would be best to represent as hex. It is common practice to do so when we are talking about multiple bits.